src/components/CustomFileManager.vue

```diff
@@ -6,19 +6,67 @@
         <girder-search @select="searchInput" hide-search-icon>
           <template #searchresult="item">
             {{ renderItem(item) }}
-            <v-icon class="mr-2">{{ iconToMdi(iconFromItem(item)) }}</v-icon>
-            {{ item.name }}
-            <div class="d-flex flex-wrap">
+            <div class="d-flex align-center" style="width: 100%">
+              <v-icon class="mr-2">{{ iconToMdi(iconFromItem(item)) }}</v-icon>
+              <span>{{ item.name }}</span>
+              <!-- First chip only -->
               <v-chip
-                small
-                v-for="(chipItem, i) in debouncedChipsPerItemId[item._id]"
-                :key="'chip ' + i + ' item ' + item._id"
-                class="ma-1"
-                v-bind="chipItem"
+                v-if="debouncedChipsPerItemId[item._id]?.chips?.[0]"
+                x-small
+                class="ma-1 type-indicator"
+                v-bind="debouncedChipsPerItemId[item._id]?.chips?.[0]"
                 @click.stop
               >
-                {{ chipItem.text }}
+                {{ debouncedChipsPerItemId[item._id]?.chips?.[0].text }}
               </v-chip>
+              <v-chip
+                v-else-if="computedChipsIds.has(item._id)"
+                x-small
+                class="ma-1 type-indicator"
+                color="grey darken-1"
+              >
+                Loading info...
+              </v-chip>
+              <span class="text-caption grey--text mx-2">
+                <v-tooltip right>
+                  <template v-slot:activator="{ on, attrs }">
+                    <span v-bind="attrs" v-on="on">
+                      Modified: {{ formatDateString(item.updated) }}
+                    </span>
+                  </template>
+                  Created: {{ formatDateString(item.created) }}
+                </v-tooltip>
+              </span>
```

Comment:
This tooltip can be made using our custom tooltip directive:
```
<span
      class="text-caption grey--text mx-2"
      v-tooltip="{
        content: `Created: ${formatDateString(item.created)}`,
        position: 'right',
      }"
    >
      Modified: {{ formatDateString(item.updated) }}
</span>
```

Reply:
(You will need to increase NimbusTooltip.vue z-index to at least 10001 to let it be above the GirderSearch component)